<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chooseDialog</class>
 <widget class="QWidget" name="chooseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>149</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chose a directory to locate your suite files..</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="chooseLineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>Choose a directory to locate your suite files...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="chooseButton">
       <property name="text">
        <string>Choose</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="okbutton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
